docs/appendix/troubleshooting.md

```diff
@@ -158,6 +158,68 @@ It will resemble the following:
 }
 ```
 
+### Error: failed to find camera
+
+**Additional Errors:** `cannot open webcam`, and `found no webcams`.
+
+**Description:** When working with a [camera](/components/camera/) component on the Linux platform, your Linux OS must be able to access the camera properly, and the camera must be configured to use a pixel format that Viam supports.
+
+**Solution:** On your Linux system, verify each of the following:
+
+- Ensure that your Linux OS is able to access your camera:
+
+  1.  First, find your video device:
+
+      ```sh {class="command-line" data-prompt="$"}
+      ls -l /dev/v4l/*
+      ```
+
+      In the list of camera devices returned, find the entry that matches the `video_path` from your camera component's configuration in the Viam app.
+      For example, if your camera is configured with a `video_path` of `usb-GENERAL_GENERAL_WEBCAM-video-index0`, you would find it in the output of the above command like so:
+
+      ```sh
+      usb-GENERAL_GENERAL_WEBCAM-video-index0 -> ../../video0
+      ```
+
+      The video device number is the number after `video` appearing on the symlink target for that device, in this case `0`.
+
+  1.  Then, [stop `viam-server`](/installation/manage/#run-viam-server), and verify that your Linux OS is able to access that video device properly:
+
+      ```sh {class="command-line" data-prompt="$"}
+      v4l2-ctl -d0 --stream-mmap
```

Comment:
They could also use `v4l2-ctl --stream-count 1` to try a single frame rather than looping forever. The downside is there's only output when things go wrong. For example, 
```
➜  v4l2rtspserver git:(master) v4l2-ctl --device /dev/video0 --stream-count 1       # valid device
➜  v4l2rtspserver git:(master) v4l2-ctl --device /dev/video999 --stream-count 1   # invalid device
Cannot open device /dev/video999, exiting.

```
I'm OK with whichever you think is simpler.